### PR TITLE
Yr  entity export input to pom

### PR DESCRIPTION
**Pull request by disel20**
Input Test change to POM (right now without Embedded, but will added)

### CODE REVIEW

File: src/test/java/test/entity/EntityExportTest.java [hunk 1]
```diff
@@ -80,13 +80,13 @@ public void inputTest() {
                 .fillInt(exportInt)
                 .fillDecimal(exportDec)
                 .fillDate(exportDate)
-                .fillDateTime(exportDateTime)
+                .fillDateTime(EDITED_DATETIME)
```
Comment (panteleyeva) [inline src/test/java/test/entity/EntityExportTest.java:83]: Why did you change exportDateTime to EDITED_DATETIME(this variable created for editTest)?

Comment (disel20) [reply src/test/java/test/entity/EntityExportTest.java:83]: I did it for solve with some falls test, I don't know why, but if we run both test: Input + view test we getting falling, EDITED_DATETIME variable like static and for now I choose it

**Comment by disel20** [comment]
closed because were mistakes inside
